From e9c9da64d54c2044014339dddc867f80ab90500b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=D0=90=D0=BB=D0=B5=D0=BA=D1=81=D0=B5=D0=B9?=
 <alexey.pawlow@gmail.com>
Date: Thu, 17 Jun 2021 18:52:06 +0530
Subject: [PATCH 034/N] build: remove usage of MS_COREDLL
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

it's not set by the autotools build system and deprecated,
so port it to known macros.

Co-authored-by: Алексей <alexey.pawlow@gmail.com>
---
 Python/sysmodule.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/Python/sysmodule.c b/Python/sysmodule.c
index 6ade6b3..4275c5d 100644
--- a/Python/sysmodule.c
+++ b/Python/sysmodule.c
@@ -53,7 +53,7 @@ Data members:
 #  include <windows.h>
 #endif /* MS_WINDOWS */
 
-#ifdef MS_COREDLL
+#if defined(MS_WINDOWS) && defined(Py_ENABLE_SHARED)
 extern void *PyWin_DLLhModule;
 /* A string loaded from the DLL at startup: */
 extern const char *PyWin_DLLVersionString;
@@ -3366,14 +3366,14 @@ version -- the version of this interpreter as a string\n\
 version_info -- version information as a named tuple\n\
 "
 )
-#ifdef MS_COREDLL
+#if defined(MS_WINDOWS) && defined(Py_ENABLE_SHARED)
 /* concatenating string here */
 PyDoc_STR(
 "dllhandle -- [Windows only] integer handle of the Python DLL\n\
 winver -- [Windows only] version number of the Python DLL\n\
 "
 )
-#endif /* MS_COREDLL */
+#endif /* defined(MS_WINDOWS) && defined(Py_ENABLE_SHARED) */
 #ifdef MS_WINDOWS
 /* concatenating string here */
 PyDoc_STR(
@@ -3900,7 +3900,7 @@ _PySys_InitCore(PyThreadState *tstate, PyObject *sysdict)
     SET_SYS("_is_mingw_ucrt", PyLong_FromLong(is_mingw_ucrt()));
     SET_SYS("_use_alt_sep", PyLong_FromLong(use_alt_sep()));
 
-#ifdef MS_COREDLL
+#if defined(MS_WINDOWS) && defined(Py_ENABLE_SHARED)
     SET_SYS("dllhandle", PyLong_FromVoidPtr(PyWin_DLLhModule));
     SET_SYS_FROM_STRING("winver", PyWin_DLLVersionString);
 #endif
